anchore_engine/analyzers/syft/__init__.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
```

Comment:
is this a script?

Reply:
snuck in there from the initial gems PR work (before it was an imported python module and instead being shelled out to). I'll take it out